core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -1005,6 +1005,18 @@ public static String getDarwinRegex() {
         return (tabs.length > 0 && disabledTabs.length() > 0) ? Arrays.asList(tabs) : new ArrayList<String>();
     }
     
+    public static String getDisabledTabList() {
```

Comment:
might be better to name this e.g. `getDisabledTabAsString` or something, feels weird to return String for a function named `..List()`